pkg/kotsadm/kotsadm_objects.go

```diff
@@ -109,6 +110,66 @@ func kotsadmServiceAccount(namespace string) *corev1.ServiceAccount {
 	return serviceAccount
 }
 
+func updateKotsadmDeployment(deployment *appsv1.Deployment, deployOptions types.DeployOptions) error {
+	var securityContext corev1.PodSecurityContext
```

Comment:
I don't think this is needed for this migration

Reply:
@marccampbell `kotsadmDeployment` function does this too... maybe leave it?